x-pack/plugins/security_solution/server/lib/entity_analytics/risk_score/risk_score_data_client.ts

```diff
@@ -210,6 +202,14 @@ export class RiskScoreDataClient {
         },
       });
 
+      // Delete the component template without the namespace in the name
```

Comment:
Why is this so far from the rest of the component template code? Could we put all the component template migration into a single function? The init function is getting out of control.

Reply:
I moved the creation and renaming of the component template into a new function and positioned the deletion immediately after updating the index template to reference the new component template.